module1/presentation_homework.md

```diff
@@ -2,35 +2,15 @@
 
 # Podstawy C++
 
-## Podsumowanie
+## Praca domowa
 
 <a href="https://coders.school">
     <img width="500px" data-src="../coders_school_logo.png" alt="Coders School" class="plain">
 </a>
 
-___
-
-## Co pamiętasz z dzisiaj?
-
-### Napisz na czacie jak najwięcej haseł
-<!-- .element: class="fragment fade-in" -->
-
-1. <!-- .element: class="fragment fade-in" --> Typy danych
-    * `void`, `bool`, `char`, `int`, `double` + ich odmiany
-1. <!-- .element: class="fragment fade-in" --> Funkcje
-    * sygnatura (deklaracja) = typ zwracany, nazwa, argumenty
-1. <!-- .element: class="fragment fade-in" --> Instrukcje warunkowe
-    * `if`, `switch/case`
-1. <!-- .element: class="fragment fade-in" --> Pętle
-    * `for`, `while`, `do/while`
-1. <!-- .element: class="fragment fade-in" --> Tablice
-    * `Type t[N]`, `operator[]`
-
```

Comment:
Może jednak zostawmy taką powtórkę też w modułach. Nie wiem czy raz nie mówię w jedną stronę a raz w drugą :D  Ale w sumie podsumowań nigdy za wiele. Najlepiej niech będzie tutaj oraz w intro/outro. Także ten plik przywróć do poprzedniej wersji i może tego się trzymajmy ;) Powtórki mogą być zduplikowane, to w procesie nauczania nie zaszkodzi.